### DISCUSSION TITLE
Am i able to control which ZOOM levels the map can use when using the mouse scrollwheel?

### DISCUSSION THREAD

**Discussion by kreativejuices** [openlayers/openlayers#15182] [Q&A] [answered]
Right now its being allowed to "zoom" in tiny increments thus having quite a few blurry stages before it jump to the next base map tile set. I want to be able to specify exactly what zoom levels the mouse scroll wheel is allowed to trigger :)

**Comment by prosperva** [comment]
Look into this

`map.events.on({ "zoomend": function(){
    //Do whatever you need to do here
}});
`

**Reply by kreativejuices** [reply]
I didn't know that event existed, i can make use of that in another issue! thanks

**Reply by ahocevar** [reply]
This was OpenLayers 2, so don't bother.

**Reply by kreativejuices** [reply]
thanks man!

**Comment by ahocevar** [comment, marked as answer]
You'll want to configure your view with `constrainResolution: true`.

**Reply by kreativejuices** [reply]
Perfect! Thank you!
